### DISCUSSION TITLE
You've used 90% of your included Automations limit

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8981] [Help] [answered]
Hi there

I got a message (/mail) that I've used 90% of my included Automations limit ... but I have not been using my Budibase Cloud environment for the last couple of week!? How/where can I check what's going wrong?

Thanks in advance, and best regards, Dick

**Comment by melohagan** [comment, marked as answer]
Hey @ecxs 

Do you have (now or previously) any Cron trigger automations? What is your tenant id?

**Reply by ecxs** [reply]
Hi @melohagan not that I'm aware off ... my tenant id = ecorganizer

**Reply by melohagan** [reply]
And if you click on the **Usage** tab within the main Budibase portal, how many automations does it show have been used?
Example:
![Screenshot 2022-12-09 at 13 51 48](https://user-images.githubusercontent.com/101575380/206717347-bfdac549-4d02-4e61-a716-c92d6e300632.png)

**Reply by ecxs** [reply]
![Screenshot 2022-12-09 at 14 53 59](https://user-images.githubusercontent.com/4005405/206717809-3a6ce40b-2176-4de4-b753-8c13224b23a0.png)

**Reply by melohagan** [reply]
Thanks @ecxs 
Could you also go through the [Automation history](https://docs.budibase.com/docs/logs) for each of your apps please. Just in case there is some rogue automation still running.

**Comment by ecxs** [comment]
@melohagan already did that ... they're all empty ...

![Screenshot 2022-12-09 at 15 07 09](https://user-images.githubusercontent.com/4005405/206720266-df1f2bc1-4430-4110-b1b1-ee81a48712ac.png)

**Reply by ecxs** [reply]
@melohagan issue remains ...
<img width="568" alt="Screenshot 2023-01-09 at 08 58 00" src="https://user-images.githubusercontent.com/4005405/211262463-58e9126e-33c3-487c-93fd-fed6f8e69b59.png">

**Reply by shogunpurple** [reply]
Hi @ecxs - it appears that you do have a running cron automation on one of your apps. We just checked the production database in cloud and can see it. It's an automation running with a call to zapier, and it's burning through your automation credit. The app name is `ContentApprovalSystem`. You should find the cron automation in there, which you should be able to delete and republish.

**Reply by ecxs** [reply]
@shogunpurple thank for your help, and please apologise for having totally overlooked this cron automation
